### ISSUE TITLE
Importing/Calling models from sktime

### ISSUE THREAD

**Issue by yttuncel** [timeseriesAI/tsai#486]
Hi @oguiza!

I'm comparing some deep learning models with other TSC machine learning algorithms over various Freezing of Gait datasets (a common symptom seen in Parkinson's Disease patients). I'm interested in some of the methods implemented in the sktime package (e.g. shapelet based, HIVECOTE, DrCIF to name a few). 

Is there a way to call these classifiers from within tsai? or do I just call them through their sklearn-like interface (.fit() etc.)?
The reason I possibly wouldn't prefer that is because I use the batch transforms and some other helper functions available to dataloaders.

Thanks!
Yigit

**Comment by oguiza** [comment]
Hi @yttuncel,
I understand your comment, but unfortunately it’s not possible to use sktime models from tsai, and there are no plans for any further integration in the future. 
sktime is a great library focused on machine learning models while tsai is focused on deep learning models. They have a different API, tsai is written in Pytorch, etc.
There’s a family of models though ([Rocket](https://timeseriesai.github.io/tsai/models.ROCKET.html) and [MiniRocket](https://timeseriesai.github.io/tsai/models.MINIROCKET.html)) that can be used in tsai. These were added because they are state of the art models that are very fast and can even be used with a CPU. They only work with small sample sets (<10k samples). We added them to have a quick way to establish a good baseline. But they have a special, sklearn-like interface, and don’t accept tsai’s transforms. You actually need to install sktime to use these models with tsai.
